lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -193,6 +193,12 @@ class CategoryRenderer {
     const descriptionEl = this._dom.createChildOf(element, 'div', 'lh-perf-hint__description');
     descriptionEl.appendChild(this._dom.convertMarkdownLinkSnippets(audit.result.helpText));
 
+    if (audit.result.debugString) {
```

Comment:
it was, but not if it didn't completely fail (this was sort of drive by, there are other debugStrings that can be set)